config.yml

```diff
@@ -0,0 +1,6 @@
+jtype: ImageTorchEncoder
+with:
+  {}
+metas:
+  py_modules:
+    - image_torch_encoder.py
```

Comment:
this seems not consistent with you file name? Which is `encoder.py`

Reply:
Thanks, fixed.